File: server/iih_pub/iih.ci_pub/src/main/java/iih/ci/rcm/hospitalinfectiondto/d/HospitalInfectionDTO.java
package iih.ci.rcm.hospitalinfectiondto.d;

import xap.mw.core.data.*;
import xap.mw.coreitf.d.*;
import java.math.BigDecimal;

/**
 * 实体 DTO数据 
 * 
 */
public class HospitalInfectionDTO extends BaseDTO {
	private static final long serialVersionUID = 1L;
	
	/**
	 * 名称
	 * @return String
	 */
	public String getName() {
		return ((String) getAttrVal("Name"));
	}
	/**
	 * 名称
	 * @param Name
	 */
	public void setName(String Name) {
		setAttrVal("Name", Name);
	}
	/**
	 * 节点id
	 * @return String
	 */
	public String getId_node() {
		return ((String) getAttrVal("Id_node"));
	}
	/**
	 * 节点id
	 * @param Id_node
	 */
	public void setId_node(String Id_node) {
		setAttrVal("Id_node", Id_node);
	}
	/**
	 * 父节点id
	 * @return String
	 */
	public String getId_parent_node() {
		return ((String) getAttrVal("Id_parent_node"));
	}
	/**
	 * 父节点id
	 * @param Id_parent_node
	 */
	public void setId_parent_node(String Id_parent_node) {
		setAttrVal("Id_parent_node", Id_parent_node);
	}
	/**
	 * 类型
	 * @return String
	 */
	public String getStyle() {
		return ((String) getAttrVal("Style"));
	}
	/**
	 * 类型
	 * @param Style
	 */
	public void setStyle(String Style) {
		setAttrVal("Style", Style);
	}
	/**
	 * 是否主卡
	 * @return FBoolean
	 */
	public FBoolean getIs_main_card() {
		return ((FBoolean) getAttrVal("Is_main_card"));
	}
	/**
	 * 是否主卡
	 * @param Is_main_card
	 */
	public void setIs_main_card(FBoolean Is_main_card) {
		setAttrVal("Is_main_card", Is_main_card);
	}
	/**
	 * 是否副卡
	 * @return FBoolean
	 */
	public FBoolean getIs_vice_card() {
		return ((FBoolean) getAttrVal("Is_vice_card"));
	}
	/**
	 * 是否副卡
	 * @param Is_vice_card
	 */
	public void setIs_vice_card(FBoolean Is_vice_card) {
		setAttrVal("Is_vice_card", Is_vice_card);
	}
	/**
	 * 报卡状态
	 * @return String
	 */
	public String getReport_status() {
		return ((String) getAttrVal("Report_status"));
	}
	/**
	 * 报卡状态
	 * @param Report_status
	 */
	public void setReport_status(String Report_status) {
		setAttrVal("Report_status", Report_status);
	}
}

File: client/iih_stub/iih.ci_stub/iih.ci.rcm/hospitalinfectiondto/d/HospitalInfectionDTO.cs
using System;
using xap.mw.core.data;
using xap.mw.coreitf.d;

namespace iih.ci.rcm.hospitalinfectiondto.d
{
    /// <summary>
    /// HospitalInfectionDTO 的摘要说明。
    /// </summary>
    public class HospitalInfectionDTO : BaseDTO {

        public HospitalInfectionDTO() {
 
        }

        /// <summary>
        /// 名称
        /// </summary>
		public string Name {
            get { return getAttrVal<string>("Name",null); }
            set { setAttrVal<string>("Name", value); }
        }

        /// <summary>
        /// 节点id
        /// </summary>
		public string Id_node {
            get { return getAttrVal<string>("Id_node",null); }
            set { setAttrVal<string>("Id_node", value); }
        }

        /// <summary>
        /// 父节点id
        /// </summary>
		public string Id_parent_node {
            get { return getAttrVal<string>("Id_parent_node",null); }
            set { setAttrVal<string>("Id_parent_node", value); }
        }

        /// <summary>
        /// 类型
        /// </summary>
		public string Style {
            get { return getAttrVal<string>("Style",null); }
            set { setAttrVal<string>("Style", value); }
        }

        /// <summary>
        /// 是否主卡
        /// </summary>
		public bool? Is_main_card {
            get { return getAttrVal<FBoolean>("Is_main_card",null); }
            set { setAttrVal<FBoolean>("Is_main_card", value); }
        }

        /// <summary>
        /// 是否副卡
        /// </summary>
		public bool? Is_vice_card {
            get { return getAttrVal<FBoolean>("Is_vice_card",null); }
            set { setAttrVal<FBoolean>("Is_vice_card", value); }
        }

        /// <summary>
        /// 报卡状态
        /// </summary>
		public string Report_status {
            get { return getAttrVal<string>("Report_status",null); }
            set { setAttrVal<string>("Report_status", value); }
        }
        /// <summary>
        /// 返回属性列表
        /// </summary>
        /// <returns></returns>
        public override string[] getAttrNames()
        {
            return new string[]{ "Name", "Id_node", "Id_parent_node", "Style", "Is_main_card", "Is_vice_card", "Report_status"};
        }
        
        /// <summary>
        /// 返回全路径DO类名
        /// </summary>
        /// <returns></returns>
        public override string getFullClassName()
        {
            return "iih.ci.rcm.hospitalinfectiondto.d.HospitalInfectionDTO";
        }
    }
}
